internal/cri/server/container_create.go

```diff
@@ -1027,6 +1027,11 @@ func (c *criService) buildWindowsSpec(
 		annotations.DefaultCRIAnnotations(sandboxID, containerName, imageName, sandboxConfig, false)...,
 	)
 
+	if config.Windows != nil && config.Windows.Resources != nil &&
```

Comment:
Perhaps move this to something like `setWindowsResources` to split up this multi-if block